unlock-app/src/components/content/events-collection/EventDetailDawer.tsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react'
+import { Event } from './EventsCollectionDetailContent'
+import { Drawer, Placeholder } from '@unlock-protocol/ui'
+import { AiOutlineCalendar as CalendarIcon } from 'react-icons/ai'
+import { AiOutlineGlobal as GlobeIcon } from 'react-icons/ai'
+import { useCheckoutConfig } from '~/hooks/useCheckoutConfig'
+import { RegistrationCard } from '../event/Registration/RegistrationCard'
+import { useEventOrganizers } from '~/hooks/useEventOrganizers'
+import Hosts from '../event/Hosts'
+import { EventDetail } from '../event/EventDetail'
+import { EventLocation } from '../event/EventLocation'
+
+interface EventDetailDrawerProps {
+  isOpen: boolean
+  setIsOpen: (isOpen: boolean) => void
+  event: Event | null
+}
+
+// Helper function to get ordinal suffix for a given day
+const getOrdinalSuffix = (day: number): string => {
```

Comment:
There must be a library that does this, no? We already use `dayjs` and I suspect it support this!